azure-iot-device/azure/iot/device/aio/abomination.py

```diff
@@ -0,0 +1,265 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""This module provides hard coded patches used to modify items from the libraries.
+Currently we have to do like this so that we don't use exec anywhere"""
+
+
+def execute_atrocity_for_async():
```

Comment:
I'd prefer if we could keep the atrocities and abominations to one file. The original patch.py worked for both due to the underscore being added to the class name (as you can see here, still copied from the original's output). I don't see why we couldn't do both here as well.

Reply:
yes precisely u guessed the reason. It was in 1 file but cant be..

---
In reply to: [389205328](https://github.com/Azure/azure-iot-sdk-python/pull/482#discussion_r389205328) [](ancestors = 389205328)